_posts/en/2020-07-28-micro-procurment-whats-stopping-us.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "Micro-procurement for digital work - what's stopping us?"
+ref: working-in-the-open-p1
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team, and procurement experts at ESDC and TBS"
+date: "2020-07-28"
+excerpt_separator: <!--more-->
+
+---
+
+The Government of Canada (GC) buys a lot of stuff. In 2016, the total spent by the GC on goods and services was ~$15.9B. From an IT point
+of view, this spending can include big ticket items like new data centres and new national applications costing tens of millions of
+dollars, but can also include low dollar value items like paying someone to build a small prototype or write a position paper.
+
+<!--more-->
+
+To ensure that purchasing is working well across the GC, procurement teams are working hard on modernization efforts including the
+development of an [electronic procurement system](https://buyandsell.gc.ca/initiatives-and-programs/the-electronic-procurement-solution/
+about), enabling agile procurement, [social procurement pilots](https://buyandsell.gc.ca/procurement-data/tender-notice/PW-18-00828087)
+and [contract language simplification](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/simpcont-simpcont-eng.html). Interestingly, Low Dollar
+Value (LDV) procurements don't seem to make the list for procurement modernization priorities. This could be because these procurements
+are low risk and generally don't make headline news. However for this same reason they provide the perfect opportunity for procurement and IT experimentation.
+
+Imagine that you manage an IT product and a team member notices a bug in one of the open source components, but doesn't have time to fix it. What if you could post an opportunity for
+someone to fix that bug for $2K, select someone to do the work and receive the fixed and tested code within a matter of days?
```

Comment:
Is there a way to indent is and format it differently? I dont think it fits in the flow of the document here, but I think it's a good excerpt to add.

Reply:
I wondered about that too. I don't think blockquote works. And using a table to format is a no. I could try doing something in css? Otherwise I could make it all italics to make it look different?

Reply:
I reformatted the document :)

Reply:
Fixed but I had to use html. Means I get error messages but I could change the file from .md to .html to get rid of that (like we did with the teams page). Let me know what you think.

Reply:
I think the new class works well, should it be with a different background as well, adding a bit more focus but different than for quotes?